.github/workflows/ci.yaml

```diff
@@ -32,3 +32,14 @@ jobs:
           export PATH=$PATH:/usr/local/kubebuilder/bin
       - name: Tests
         run: go test -v ./...
+      - name: E2E Tests
+        env: 
+          KUBECONFIG: ${{ github.workspace }}/kubeconfig
+        run: |
+          printf "${{ secrets.E2E_CLUSTER_SECRET }}" | base64 --decode > "${KUBECONFIG}"
+          curl -L https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64 -o kuttl;
+          chmod +x kuttl;
+          ./kuttl test;
```

Comment:
Can we also run this in the e2e nightly, against the operator that workflow sets up?